src/components/SideNav/SideNav.tsx

```diff
@@ -0,0 +1,26 @@
+import { BiFile } from 'react-icons/bi'
+
+export default function SideNav() {
+  return (
+    <div className="bg-[#03111b] w-[305px] text-[#FFFFFF]">
+      <div className="mt-[55px] ml-[56px]">
+        <div className=" gap-x-3 mb-8 text-3xl">
+          <BiFile className="inline-block  text-blue-600 mr-2"></BiFile>
+          <span className="text-2xl">Report Builder</span>
+        </div>
+        <div className="mb-8 leading-10 ">
+          <ul>
+            <li>My Reports</li>
+            <li>Analytics</li>
+            <li>Knowledge Base</li>
+            <li>Help & Support</li>
+          </ul>
```

Comment:
can move to a new component called MenuItem or Item under SideNav/components

Reply:
Cool~